prometheus-nagios-exporter.service

```diff
@@ -2,7 +2,22 @@
 Description=Prometheus Nagios Exporter Service
 
 [Service]
-ExecStart=/opt/mlab/prometheus-nagios-exporter/nagios_exporter.py --path /var/lib/nagios3/rw/live --perf_data "--data_names=check_disk_boot=used;;;;total" --whitelist nagios_check_vdlimit_iupui_ndt_perf_data --whitelist nagios_check_disk_boot_perf_data_total --whitelist nagios_check_disk_boot_perf_data_used --whitelist nagios_check_ndt_e2e_state
+ExecStart=/opt/mlab/prometheus-nagios-exporter/nagios_exporter.py \
```

Comment:
Oh, wait -- do the line continuations work as intended in systemd units? I remember these being picky. Is this confirmed to work?

Reply:
From https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Description:

> Lines ending in a backslash are concatenated with the following line while reading and the backslash is replaced by a space character. This may be used to wrap long lines.

Also, this is already running on EB. :)